extension/cart/createProductCartItemList.js

```diff
@@ -48,6 +55,14 @@ module.exports = function (context, input, cb) {
     } else {
       transformedProduct = new SimpleProduct(products[i].productId, products[i].quantity)
     }
+    if (products[i].hasOwnProperty('properties')) {
+      for (j = 0; j < products[i].properties.length; j++) {
```

Comment:
We should get rid of those oldschool for loops, please use a loop like foreach so you don't have to use temp variables like i and j to access the current item.